lib/blpapi-wrapper.ts

```diff
@@ -334,7 +348,25 @@ export class Session extends events.EventEmitter {
         });
     }
 
-    subscribe(subscriptions: Subscription[], cb?: (err: any) => void): Promise<void> {
+    request(uri: string,
+            requestName: string,
+            request: any,
+            identity: blpapi.Identity,
+            callback: IRequestCallback): void
+    {
+        this.doRequest(uri, requestName, request, callback, false, identity);
+    }
+
+    authorizeUser(request: any, callback: IRequestCallback): void {
+        var uri = '//blp/apiauth';
+        var requestName = 'AuthorizationRequest';
+        this.doRequest(uri, requestName, request, callback, true, undefined);
```

Comment:
Well, even if it were multiple chunks, we could make a change of this sort because there's only 2 possible outcomes we care about communicating (the `Identity` for success, and an error message for failure). For now I was just maintaining identical behavior (i.e. as much as possible authorization doesn't look different from other requests), but am open to switching this up if we feel that's best.